object-cache.php

```diff
@@ -118,6 +118,23 @@ function wp_cache_delete_group( $group ) {
 function wp_cache_flush() {
 	global $wp_object_cache;
 
+	if ( function_exists( 'wp_debug_backtrace_summary' ) ) {
+		$trace = wp_debug_backtrace_summary();
+	} else {
+		$backtrace = debug_backtrace( DEBUG_BACKTRACE_IGNORE_ARGS, 1 );
+		$caller    = array_shift( $backtrace );
+		$trace =  $caller['file'], $caller['line'];
+	}
+	error_log( sprintf( 'wp_cache_flush() requested from ' . $trace ) )
```

Comment:
Missing semicolon; did you test this?

Reply:
No not yet, sorry this was meant to be draft, but I wanted to check if this approach was good with you